### DISCUSSION TITLE
Arkadaşlar merhabalar. Bunların Türkçe karşılığı olsa neler olur?

### DISCUSSION THREAD

**Discussion by omerfeyyaz** [turkce-yazilim-konati/yazilimca#39] [Soru-Yanıt] [answered]
1. Float(ondalık sayı)
2. void(tanımsız)
3. Bool(DoğruYanlış)
4. Vector(Vektör)
5. Static(Statik)
6. Boolean(Çift Değer)
Arkadaşlar merhabalar Bunların Türkçe karşılığı olsa neler. Olur ? Veya siz Türkçe söz dilimli dil yazsanız. Ne yapardınız?

**Comment by volkantash** [comment, marked as answer]
Konular başlıklara ayrılmıştır. Buralardan sürdürebilirsiniz.
bool, boolean: https://github.com/turkce-yazilim-konati/yazilimca/discussions/43
vector: https://github.com/turkce-yazilim-konati/yazilimca/discussions/44
static: https://github.com/turkce-yazilim-konati/yazilimca/discussions/46
void: https://github.com/turkce-yazilim-konati/yazilimca/discussions/41
float: https://github.com/turkce-yazilim-konati/yazilimca/discussions/40

**(Bu başlık kilitlendi)**
